packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -113,6 +115,7 @@ export const getPublicEvent = async (username: string, eventSlug: string, prisma
         image: `${WEBAPP_URL}/${users[0].username}/avatar.png`,
         brandColor: users[0].brandColor,
         darkBrandColor: users[0].darkBrandColor,
+        theme: users[0].theme,
```

Comment:
It doesn't seem to handle the dynamic group booking separately. I am not sure what's the right approach, but currently dynamic group bookings always use system theme(by setting theme to `null`)

Reply:
Also, I don't see handling for `team.theme`. That got added not too long ago.

Reply:
Good point! I'll change dynamic theme to system theme indeed. Makes a lot of sense! Thanks for spotting this!